apps/block/templates/content_inner.jade

```diff
@@ -0,0 +1,30 @@
+if block.has('image') && block.get('class') !== 'Media'
+  .block-img
+    a( href= block.getSourceUrl(), target='_blank' )
+      img( src= block.resizeImage(1000, 1000, 'large') )
```

Comment:
I'm just realizing now that this `resizeImage` method doesn't do anything? I'm assuming we used to use the resizing proxy here?